public/ipython/notebook/js/outputarea.js

```diff
@@ -982,6 +986,24 @@ define([
         }
     };
 
+    OutputArea.prototype.convertSvg = function() {
+        for (var i = 0; i < this.outputs.length; i++) {
+            var o = this.outputs[i];
+            if (o.output_type == 'execute_result') {
+                if (o.data["text/html"]) {
+                    var h = o.data["text/html"];
+                    var svg = this.element.find("svg");
+                    if (svg.length) {
+                        var s = svg.get(0);
```

Comment:
Yep... only one will be handled... I know
It's a best effort atm, I'm waiting for feedback (based on real cases ^^)

The problem is for other kind of output, not svg I mean like dynatables...